### ISSUE TITLE
On manually deleting the selected option the initial value is still submitted. 

### ISSUE THREAD

**Issue by lukefisher10** [alphagov/accessible-autocomplete#205]
Hi Team

When a user has selected an option from the autocomplete, if they press the delete key until the input field is blank and then submit the form, the initial value is submitted, as opposed to a blank field.

**Comment by tvararu** [comment]
Yeah, this is a legit issue with the way the autocomplete works currently. Because it uses a select element underneath, unless that select element has a default option which doesn't pass validation, the user will always submit whatever is default. You need to add custom validation to prevent this.

I've marked this as a bug as we should find a better way to communicate or work around this behaviour.

**Comment by stevelacey** [comment]
Also worth noting that this also applies when entering gibberish into the field, basically anything that doesn't match an `option` results in the previously selected option being submitted.

**Comment by edwardhorsford** [comment]
My expectation is that whatever's entered in the field be submitted.

Where it's progressively enhanced from a select, this should be blank or an option - as we shouldn't allow users to leave text in the field that isn't an option.

Where it's an enhanced text box, we should leave whatever they type and submit it.

**Comment by hannalaakso** [comment]
This is a legitimate bug that needs fixing but it's also related to the wider discussion here: https://github.com/alphagov/accessible-autocomplete/issues/260

**Comment by edwardhorsford** [comment]
I wonder if it would be hard to make the autocomplete swap out the select for an input when it initialises - ie don't use selects under the hood after initialisation - that way whatever the input visually contains _is_ what gets submitted.
